conans/test/unittests/client/generators/compiler_args_test.py

```diff
@@ -100,8 +100,8 @@ def gcc_test(self):
         settings.build_type = "Release"
         gcc = GCCGenerator(conan_file)
         # GCC generator ignores the compiler setting, it is always gcc
```

Comment:
I don't know what's the story behind of this, but it's basically:
1. useless
2. incorrect